chart/openfaas/README.md

```diff
@@ -67,12 +56,20 @@ Now deploy OpenFaaS from the helm chart repo:
 helm repo update \
  && helm upgrade openfaas --install openfaas/openfaas \
     --namespace openfaas  \
-    --set basic_auth=true \
-    --set functionNamespace=openfaas-fn
+    --set functionNamespace=openfaas-fn \
+    --set basicAuthAutogen=true 
 ```
 
 > The above command will also update your helm repo to pull in any new releases.
 
+Retrieve the OpenFaaS credentials with:
+
+```sh
+USER=$(kubectl -n openfaas get secret basic-auth -o jsonpath="{.data.basic-auth-user}" | base64 --decode) && \
+PASSWORD=$(kubectl -n openfaas get secret basic-auth -o jsonpath="{.data.basic-auth-password}" | base64 --decode) && \
```

Comment:
This should match k3sup and the troubleshooting docs page, if it does then I think that's fine :+1: 

https://github.com/alexellis/k3sup/blob/f5390f692656ec5230ae9448d010e38b62d4d2ed/pkg/cmd/openfaas_app.go#L237

Reply:
It is

Reply:
Great